src/tests/ftest/harness/advanced.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from random import choice
+from re import findall
+
+from apricot import TestWithServers
+from general_utils import run_pcmd
```

Comment:
(pylint-syntax-error) Cannot import 'general_utils' due to syntax error 'invalid syntax (<unknown>, line 888)'